book.common/src/main/java/com/happystays/book/common/dto/CreditCard.java

```diff
@@ -0,0 +1,13 @@
+package com.happystays.book.common.dto;
+
```

Comment:
can you create a folder successEventModels and keep all models of Booking success event inside that